### ISSUE TITLE
Tablesorter not printing with columnSelector rows

### ISSUE THREAD

**Issue by Kezz101** [Mottie/tablesorter#808]
Hello,

I'm using the following Javascript to create my tablesorter table but when I try to print the table it works fine but the whole table is printed rather than just the ones selected with `columnSelector`.

```javascript
        $(".tablesorter").tablesorter({
            theme: "bootstrap",
            widthFixed: false,
            headerTemplate: "{content} {icon}",
            sortList: [[0,0]],
            widgets: ["filter", "zebra", "print", "columnSelector"],
            widgetOptions: {
                zebra: ["even", "odd"],
                filter_reset: ".reset",
                columnSelector_mediaquery: false,
                columnSelector_saveColumns: false,
                print_columns: "selected",
                print_callback: function(config, $table, printStyle) {
                    config.print_columns = "selected";

                    $table.find(".tablesorter-infoOnly").remove();
                    $table.find(".print-remove, .tablesorter-pager").remove();
                    $table.find(".tablesorter-childRow").remove();
                    $.tablesorter.printTable.printOutput(config, $table.html(), printStyle);
                }
            }
        });
    });
```

The whole HTML of the website is here: http://pastebin.com/zGf85LBQ

**Comment by Mottie** [comment]
Hi @Kezz101!

That looks like a bug alright... it's because the `columnSelector_mediaquery` is `false` and the `auto` checkbox is defaulting to `true` so it thinks that all columns are to be printed.

I just fixed it in the working branch. Thanks for reporting it!

Interesting side note: I just discovered that the column selector wasn't working in the [print widget demo](http://mottie.github.io/tablesorter/docs/example-widget-print.html) in Firefox; it's because the markup is wrapped in a `button` element  instead of a `div` ([see the issue here](http://jsfiddle.net/2tt25hbd/)).
